### PR TITLE
python-idna: Add missing dependency on python(3)-codecs

### PR DESCRIPTION
**Pull request by daniel-santos**
This error is exposed when using python-minimal.

Signed-off-by: Daniel Santos <daniel.santos@pobox.com>

Maintainer: Jeffery To <jeffery.to@gmail.com>, Alexandru Ardelean <ardeleanalex@gmail.com>
Compile tested: ramips, mt7620, v18.06.1
Run tested: ramips, mt7620, v18.06.1

Description:

### CODE REVIEW

File: lang/python/python-idna/Makefile [hunk 1]
```diff
@@ -36,14 +36,14 @@ endef
 define Package/python-idna
 $(call Package/python-idna/Default)
   TITLE:=python-idna
-  DEPENDS:=+PACKAGE_python-idna:python-light
+  DEPENDS:=+PACKAGE_python-idna:python-light +python-codecs
```
Comment (cshoredaniel) [inline lang/python/python-idna/Makefile]: Please use ``PACKAGE_python-idna:python-codecs``

File: lang/python/python-idna/Makefile [hunk 2]
```diff
@@ -36,14 +36,14 @@ endef
...
   VARIANT:=python
 endef
 
 define Package/python3-idna
 $(call Package/python-idna/Default)
   TITLE:=python3-idna
-  DEPENDS:=+PACKAGE_python3-idna:python3-light
+  DEPENDS:=+PACKAGE_python3-idna:python3-light +python3-codecs
```
Comment (cshoredaniel) [inline lang/python/python-idna/Makefile]: Please use ``PACKAGE_python3-idna:python3-codecs``

**Comment by daniel-santos** [comment]
I've updated with your requests and also bumped the copyright date.  I should note however that I've tested this patch against the openwrt-18.06 branch which is building v2.6 of the software and not v2.8 as in the development HEAD.

**Comment by daniel-santos** [comment]
This would be a good one to backport to openwrt-18.06.  btw, my project is not moving to v2.8 yet.

**Comment by jefferyto** [comment]
@daniel-santos I was waiting for this PR to be merged before backporting, but you can open a new PR for the openwrt-18.06 branch if you want :wink:

Oops, just noticed this: Can you increment `PKG_RELEASE`?

**Pull request merged**